docs/_sources/documentation.rst.txt

```diff
@@ -123,33 +169,13 @@ The images you want to segment must be stored following a particular architectur
     ---- pixel_size_in_micrometer.txt (*)
     ...
 
-.. NOTE ::
-   The images must be saved in **png format**. You don't have to specifically name them.
 
 * The image file *image.png* is the image to segment.
 * The file *pixel_size_in_micrometer.txt* contains a single float number corresponding
 to the resolution of the image, that is the **size of a pixel in micrometer**.
 
 .. NOTE ::
-   You can also specify the pixel size as an argument to our software (see next section).
-
-
-
-Using AxonDeepSeg
--------------------------------------------------------------------------------
-
-We provide a simple `Jupyter notebook <https://github.com/neuropoly/axondeepseg/blob/master/notebooks/getting_started.ipynb>`_ which explains how to use AxonDeepSeg for segmenting axons and myelin. You can directly launch it by using the `Binder link <https://mybinder.org/v2/gh/neuropoly/axondeepseg/master?filepath=notebooks%2Fgetting_started.ipynb>`_.
-
-To learn to use AxonDeepSeg, you will need some images to segment. If you don't have some,
-you can download the test data using the instructions in the `Data <https://neuropoly.github.io/axondeepseg/documentation.html#data>`_ section of this tutorial.
-
-.. WARNING ::
-   The current models available for segmentation are trained for patches of 512x512 pixels. This means that your input image(s) should be at least 512x512 pixels in size **after the resampling to the target pixel size of the model you are using to segment**. 
-
-   For instance, the TEM model currently available has a target resolution of 0.01 micrometers per pixel, which means that the minimum size of the input image (in micrometers) is 5.12x5.12.
-
-   **Option:** If your image to segment is too small, you can use padding to artificially increase its size (i.e. add empty pixels around the borders).
-
+   You can also specify the pixel size as an argument to our software.
```

Comment:
no need to put a note there, it is already clearly explained in the single-image example